int16[] sns_1_Indiv
int16[] sns_2_Indiv
int16[] sns_3_Indiv
